src/views/validated-designs/index.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Head from 'next/head'
+import BaseLayout from 'layouts/base-layout'
+import MobileMenuLevelsGeneric from 'components/mobile-menu-levels-generic'
+import { ProductSlug } from 'types/products'
+
+/**
+ * Stub interfaces for how we may model the data for this view
+ * Can be updated and adjusted as anyone sees fit based on the needs of the view
+ */
+export interface ValidatedDesignsLandingProps {
+	title: string
+	description: string
+	categoryGroups: HvdCategoryGroup[]
+}
+
+interface HvdCategoryGroup {
+	slug: string
+	title: string
+	description: string
+	product: ProductSlug
+	guides: HvdGuide[]
+}
+
+interface HvdGuide {
+	slug: string
+	title: string
+	description: string
+	href: string
+}
```

Comment:
Definitely not married to these types, but they were my interpretation of the view's needs based on the current designs.

Reply:
The only thing that kind of pops into my head here is to also add the file path for the HvdGuide and HvdCategoryGroup, but I'm unsure how useful that would be currently. I'm gonna need to play with this PR a bit to figure out what changes I feel like are needed. But I do think this is a great start and follows a "List -> Detail" data pattern.